integration/tests/cook/util.py

```diff
@@ -869,14 +869,15 @@ def query_queue(cook_url):
     return session.get(f'{cook_url}/queue')
 
 
-def set_limit(cook_url, limit_type, user, mem=None, cpus=None, gpus=None, jobs=None, reason='testing'):
+def set_limit(cook_url, limit_type, user, mem=None, cpus=None, gpus=None, count=None, reason='testing'):
```

Comment:
Seems like a docstring would be helpful here

Reply:
Added